tests/unit/python/fledge/common/test_configuration_manager.py

```diff
@@ -1632,7 +1880,13 @@ def mock_coro():
         category_name = 'catname'
         item_name = 'itemname'
 
-        attrs = {"query_tbl_with_payload.return_value": mock_coro()}
+        # Changed in version 3.8: patch() now returns an AsyncMock if the target is an async function.
+        if sys.version_info.major == 3 and sys.version_info.minor >= 8:
+            _attr = await mock_coro()
```

Comment:
Class dict does not define await, so the await operator cannot be used on its instances.
Remove asyncio.coroutine from the mock_coro def and make it as async

Reply:
Marking async def x is good and clean. But that marker itself make it async so kept as is. This will make it behave like future task hence async await paradigm; else it would have complained as not awaitable

Reply:
`async def` is a new syntax from Python 3.5. Therefore for practical perspective just never use @coroutine if your Python is 3.5+.
From Python 3.5 coroutines formally became a distinct type and thus the async def syntax, along with await statements.